README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML... I would say that Semantic HTML is all about the arrangements of your html in different categories that make it easier to read and easier to style. This enables the browser to display them in the way you desire as the developer.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+display block creates a block element while display inline creates an inline element. inline will take as much space as it can in a line but block doesnt try to maximize the space
+
 3. What are the 4 areas of the box model?
 
+content, margin, padding and border
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+In the cross axis. i.e for example, if the direction was row, it will make alignment in the column axis.
```

Comment:
Okay simply put, the align-items work on the cross-axis

for flex-direction: row,
main axis = horizontal
cross axis = vertical

for flex-direction: column,
main axis = vertical
cross axis = horizontal